tests/Domain/DonationRepositoryTest.php

```diff
@@ -523,16 +521,16 @@ public function testFindReadyToClaimGiftAid(): void
         $queryBuilderProphecy->where('d.donationStatus = :claimGiftAidWithStatus')
             ->shouldBeCalledOnce()->willReturn($queryBuilderProphecy->reveal());
 
-        // 5 `andWhere()`s in all, excluding the first `where()` but including the one
+        // 6 `andWhere()`s in all, excluding the first `where()` but including the one
         // NOT for `$withResends` calls.
         $queryBuilderProphecy->andWhere(Argument::type('string'))
-            ->shouldBeCalledTimes(5)->willReturn($queryBuilderProphecy->reveal());
+            ->shouldBeCalledTimes(6)->willReturn($queryBuilderProphecy->reveal());
```

Comment:
Is it worthwhile to be counting the `andWheres` etc? I'm not sure how much it really helps us understand what the code's doing or avoid bugs. Maybe worth deleting this test and just testing the DonationRepository through in integration with the DB? This feels more like coding in duplicate than coding and testing.